mods/cnc/maps/gdi02/gdi02.lua

```diff
@@ -80,7 +80,7 @@ WorldLoaded = function()
 
 	nodObjective = enemy.AddPrimaryObjective("Destroy all GDI troops.")
 	gdiObjective1 = player.AddPrimaryObjective("Eliminate all Nod forces in the area.")
-	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium Refinery.")
+	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium refinery.")
```

Comment:
https://github.com/OpenRA/OpenRA/blob/e7a64ffec4b12a4f6d7807151f6649149587c6c6/mods/cnc/rules/structures.yaml#L152 says that `Refinery` is correct.